cmd/state/state.go

```diff
@@ -219,6 +227,9 @@ func getFlags(defaultFlags GlobalFlags, env map[string]string, args []string) Gl
 	if val, ok := env["K6_BUILD_SERVICE_URL"]; ok {
 		result.BuildServiceURL = val
 	}
+	if val, ok := env["K6_BINARY_CACHE"]; ok {
+		result.BinaryCache = val
+	}
```

Comment:
```suggestion
```
I don't think we have to introduce this option. Go already provides a way to override the OS cache dir, if they really need it. However, even if it would be required, I would prefer if we wait for a real feature-request instead of introducing it now.

Reply:
Elaborating on my previous comment, the cache dir we are discussing is for storing the custom binaries downloaded from the build service, there's no go involved (users don't need to have it). Moreover, if in the future we support local builds, these two caches will be different